src/index.html

```diff
@@ -11,12 +11,107 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+          class="logo-img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="list-item">
+            <a
+              class="is-active item-link"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              class="item-link"
+              href="#"
```

Comment:
The 'href' attribute is currently set to '#', which is a placeholder. Consider updating it to the actual URL if available.